### DISCUSSION TITLE
I can't limit the results get from a relationship

### DISCUSSION THREAD

**Discussion by rimoi** [api-platform/api-platform#1768] [Q&A]
Hi!

I can't limit the results get from a relationship.

I Explain my problem, I have a relationship like this:
```php
 /**
  * @ApiResource(
  *     itemOperations={
  *          "get"={"normalization_context"={"groups"={ "user:item:get"}}}
  *     }
  */
class User
{
    // .... others code

   /**
    * @ORM\OneToMany(targetEntity=Comment::class, mappedBy="user", orphanRemoval=true)
    * @Groups({"user:item:get"})
    */
     private $comments;

      // .... others code
}
```

Let's say that the user whose `Id = 199` has  `50` comments.

I want when I call this Uri : `http://localhost/api/user/199`, I get a maximum `3` comments and no more per request,

I don't know how to do that someone has an idea?

Thanks

**Comment by maciej-laskowski** [comment]
You can change the number of items per page globally and for each entity

https://api-platform.com/docs/core/pagination/#changing-the-number-of-items-per-page
